extension/src/FileFunctions.ts

```diff
@@ -12,7 +12,10 @@ export function findFiles(pattern: string, root: string): string[] {
   return fileList.sort((a, b) => a.localeCompare(b));
 }
 
-function getAllFilesRecursive(dir: string, fileList: string[] = []): string[] {
+export function getAllFilesRecursive(
```

Comment:
- Maybe add a test case

Reply:
it is being used in the tests where it removes all files from previous tests - isn't that enough? It's not run by workflow, but every time you run it locally 🙂

Reply:
Yeah might be enough I was just thinking since it's now exported it would be nice to have a unit test. But it's probably good enough as is, it's the kind of function that'll be noticed if it starts to misbehave ;)